src/modules/imageresizer/ShellExtensions/dllmain.cpp

```diff
@@ -2,10 +2,163 @@
 #include "resource.h"
 #include "ShellExtensions_i.h"
 #include "dllmain.h"
+#include <interface/powertoy_module_interface.h>
+#include <common/settings_objects.h>
+#include "Settings.h"
 
 CShellExtensionsModule _AtlModule;
 
 extern "C" BOOL WINAPI DllMain(HINSTANCE hInstance, DWORD dwReason, LPVOID lpReserved)
 {
     return _AtlModule.DllMain(dwReason, lpReserved);
 }
+
+// The PowerToy name that will be shown in the settings.
+const static wchar_t* MODULE_NAME = L"ImageResizer";
+// Add a description that will we shown in the module settings page.
+const static wchar_t* MODULE_DESC = L"<no description>";
```

Comment:
I assume we want to create a description for this? Also this should be localized not hard coded.  Can you create a task, and do that in a seperate PR?